helix-core/src/main/java/org/apache/helix/controller/stages/BestPossibleStateOutput.java

```diff
@@ -108,4 +108,12 @@ public void setPreferenceLists(String resource,
   protected boolean containsResource(String resource) {
     return _preferenceLists != null && _preferenceLists.containsKey(resource);
   }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("preference list= ").append(_preferenceLists);
+    sb.append(", resourceStateMap= ").append(_resourceStateMap);
+    return sb.toString();
+  }
```

Comment:
Are we printing/logging this? If so, I don't think that's wise since that would be a lot of text being logged.

Reply:
removed.